index.html

```diff
@@ -1,14 +1,145 @@
 <!DOCTYPE html>
 <html lang="en">
-    <head>
-        <meta charset="utf-8"> 
-        <meta name="viewport" content="width=device-width, initial-scale=1">
-        <title>My First Microverse Project</title>
-        <link rel="stylesheet" href="style.css">
-    </head>
-    <body>
-        <header>
-            <h1 class="title-header">Hello Microverse!</h1>
-        </header>
-    </body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-uWxY/CJNBR+1zjPWmfnSnVxwRheevXITnMqoEIeG1LJrdI0GlVs/9cVSyPYXdcSF"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <nav
+      class="navbar navbar-expand-lg navbar-dark fixed-top"
+      id="navbar"
+    >
+      <div class="container">
+        <div
+          class="
+            collapse
+            navbar-collapse navbar-dark
+            d-flex
+            justify-content-end
+          "
+          id="navbarNav"
+        >
+          <ul class="navbar-nav">
+            <li class="nav-item">
+              <a class="nav-link" aria-current="page" href="#welcome-section"
+                >About</a
+              >
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" aria-current="page" href="#projects">Work</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" aria-current="page" href="#contact"
+                >Contact</a
+              >
+            </li>
+          </ul>
+        </div>
+      </div>
+    </nav>
+
+    <section
+      id="welcome-section"
+      class="
+        container-fluid
+        bg-dark
+        text-white
+        d-flex
+        justify-content-center
+        align-items-center
+      "
+    >
+      <div class="text-center">
+        <h1 class="">Hey I'm Mimic</h1>
+      </div>
+    </section>
+
+    <section
+      id="projects"
+      class="container-fluid"
+    >
+      <div class="container text-center text-white py-4">
+        <h2 class="text-decoration-underline">These are some of my projects</h2>
+
+        <div class="row gx-4">
+          <div class="col-md-4">
+            <div class="project-tile card border-0">
```

Comment:
- [ ] **Required** I highly recommend adding more information to support why a change is necessary. This helps the aids in making the changes requested.
- [ ] **Optional** As much as possible, you could employ the use of emojis to express your desired changes.